packages/@okta/vuepress-site/docs/guides/sign-into-web-app-remediation/create-okta-application/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Create an Okta application
+---
+Before you can sign a user in, you need to create an Okta application that represents your web application.
+
+Start by signing in to the Okta Developer Console:
+
+<a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
+
+1. Select **Applications**, then **Add Application**. Pick **Web** as the platform. Click **Next**.
+
+2. Enter a name for your application (or leave the default value).
+
+3. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
+
+4. Next, enter values for the **Login redirect URI**. Add values for local development (such as `http://localhost:8080/authorization-code/callback`) and production (such as `https://app.example.com/authorization-code/callback`).
```

Comment:
I think we should mention that we don't require a valid RedirectURI for this scenario. It can be whatever, for example, `http://okta.com`. We only need one because is required by the API to validate the request, but we're not redirecting users anywhere.